analysis/plotting.py

```diff
@@ -141,10 +142,18 @@ def coverage_growth_plot(self, benchmark_df, axes=None):
         axes.set(ylabel='Edge coverage')
         axes.set(xlabel='Time (hour:minute)')
 
-        ticks = np.arange(
-            0,
-            snapshot_time + 1,  # Include tick at end time.
-            snapshot_time / _DEFAULT_TICKS_COUNT)
+        if self._logscale:
+            axes.set_xscale('log')
+            ticks = np.logspace(
+                np.log10(5 * 60),  # Start from minute 5.
```

Comment:
Should use constant from https://github.com/google/fuzzbench/blob/master/common/experiment_utils.py#L21 instead of 5.
our snapshot start from 15 min now.

Reply:
Done